docs/_config.yml

```
@@ -11,7 +11,11 @@ github_username: auth0
 github_repo: auth0-cli
 
 # Build settings
-theme: minima
+remote_theme: just-the-docs/just-the-docs@v0.4.0.rc3
 
-header_pages:
-  -
+search_enabled: true
+favicon_ico: "/assets/images/favicon.png"
+
+aux_links:
+  "Auth0 CLI on GitHub":
+    - "//github.com/auth0/auth0-cli"
```

```suggestion
    - "//github.com/auth0/auth0-cli"

```

Just adding a new line at the end.